hibike/hibike_message.py

```diff
@@ -57,9 +57,11 @@
   "GenericError"        : 0xFF
 }
 
+MAX_PACKET_SIZE = 64
```

Comment:
You're right that there's a bit limit.  Unfortunately, it's unidirectional.  (We tested this last night, using pyusb).

There's a 63 byte limit in packets from the HID Smart Device to the BBB.  From the BBB to the HID smart device, there seems to be no practical limit in packet size.  (We tested this up to 100 bytes).

Therefore, this limit needs to go away, and a similar limit needs to be implimented on the embedded side.